### DISCUSSION TITLE
problem with x-show on this release

### DISCUSSION THREAD

**Discussion by ravenshill** [alpinejs/alpine#2079] [7. Bugs (archived)] [answered]
x-show is not working on 3.3.4, is ok on 3.3.3 . It shows up on very simple code.  see some snippets from my code below. Have not included the full source here.  I noticed this update included some code changes for x-show.   

 <button @click="isOpen=!isOpen; console.log('Popular clicked');  console.log(isOpen);" type="button" 
<div x-show="isOpen" x-cloak @click.outside="isOpen = false"

**Comment by SimoTod** [comment]
Can you post the full snippet and enclose it in ``` otherwise github won't show it correctly

**Reply by SimoTod** [reply]
@ravenshill It seems to work correctly: https://codepen.io/SimoTod/pen/wvepxpg
There are also unit tests for simple cases so it would be weird if it was totally broken. Are you sure you don't have a typo in your code?

**Reply by ravenshill** [reply]
There has been a few comments on the issue I opened from this and others have had the same problem.

**Reply by SimoTod** [reply]
The issue in 3.3.4 is with click.away though. Is that the case for your script?

**Comment by ravenshill** [comment, marked as answer]
Fixed with 3.3.5
